yaml/csr.yaml.m4

```diff
@@ -4139,7 +4139,7 @@ ifelse(eval(CLIC != 0), 1, [[[
       description: >
           Alias for mstatus.MPP
       type: WARL
-      reset_val: 0
+      reset_val: 3
```

Comment:
Make depedend on param so that it can be merged to 40S

Reply:
The `mcause` reset value is the same for both cores?